python/py-mistune/Portfile

```diff
@@ -4,16 +4,15 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-mistune
-version             0.8.4
+version             2.0.4
 revision            0
-epoch               1
 
 categories-append   devel textproc
 supported_archs     noarch
 license             BSD
 
 python.versions     27 37 38 39 310
-python.pep517       no
+python.pep517       yes
```

Comment:
python.pep517 is not compatible with py27 variant

Reply:
py-m2r depends on py27-mistune

Reply:
Let's try to use pep517 only for Python 3.